internal/migrations/sql/021_remove_delete_cascade.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE aws_reservation_details
+DROP CONSTRAINT aws_reservation_details_pubkey_id_fkey,
+ADD CONSTRAINT aws_reservation_details_pubkey_id_fkey
+FOREIGN key (pubkey_id) REFERENCES pubkeys(id) ON DELETE SET NULL;
+ALTER TABLE aws_reservation_details ALTER COLUMN pubkey_id DROP NOT NULL;
```

Comment:
Is the double drop of `NOT NULL` here and then again in `022_pubkey_nullable.sql` intentional? I am trying to figure out why would you do that, but I feel like it is just an oversight.

Reply:
Yeah, you are right, this was not intentional. Dropping these lines.